app/views/categories/show.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.name @category.name
```

Comment:
You should be consistent in the way you set up your json. You currently have:
* pins#index -> ``` { pins: [ {id: 1, ...}, {id: 2, ...},... ] } ```
* pins#show -> ``` { pin: [ {id: 1, ...} ] } ```
* categories#index -> ``` { categories: [ {id: 1, ...}, {id: 2, ...},... ] } ```
* categories#index -> ``` {id: 1, ...} ```     (without "categories" or "category" at the top level)

---> I think the best practice (quoting [jsonapi.org](http://jsonapi.org/format/)) is to always have the same structure with the model name in plural as the key, that takes an array of 1 or more items, no matter how many instances of the model you are displaying. For pins#show, it would then be exactly similar as for index: ``` { pins: [ {id: 1, ...} ] } ```